components/Common/Sidebar/SidebarItem/index.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import type { FC } from 'react';
+
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+import type { SidebarItemType } from '..';
+
+type SideBarItemProps = SidebarItemType & {
+  isActive?: boolean;
+  onClick: (title: string) => void;
+};
```

Comment:
```suggestion
type SidebarItemProps = {
  title: string;
  url: string;
  isActive?: boolean;
};
```

Will get the activeItem from outside, there is no need for `onClick`